src/Development/IDE/Plugin/Completions.hs

```diff
@@ -153,8 +153,6 @@ getCompletionsLSP lsp ide
             let !position' = fromCurrentPosition mapping position
             pfix <- maybe (return Nothing) (flip VFS.getCompletionPrefix cnts) position'
             case (pfix, completionContext) of
-              (Just (VFS.PosPrefixInfo _ "" _ _), Just CompletionContext { _triggerCharacter = Just "."})
```

Comment:
This case prevented suggestions when typing the first dot (as in `import Control.`) because prefixText in this situation prefixText is still empty.